plugins/nginx-vhosts/install

```diff
@@ -17,6 +17,11 @@ case "$DOKKU_DISTRO" in
   arch)
     echo "%dokku ALL=(ALL) NOPASSWD:/usr/bin/systemctl reload nginx, /usr/sbin/nginx -t" > /etc/sudoers.d/dokku-nginx
     ;;
+
+  centos)
+    echo "%dokku ALL=(ALL) NOPASSWD:/usr/bin/systemctl reload nginx, /usr/sbin/nginx -t" > /etc/sudoers.d/dokku-nginx
+    echo "Defaults:dokku !requiretty" >> /etc/sudoers.d/dokku-nginx
```

Comment:
CentOS has `requiretty` by default in `/etc/sudoers` and requires that `sudo` [command has tty](https://www.shell-tips.com/2014/09/08/sudo-sorry-you-must-have-a-tty-to-run-sudo/). I was getting `sudo: sorry, you must have a tty to run sudo` error when using `git push`, dokku couldn't restart the nginx with sudo.